app/controllers/incomeSources/add/CheckForeignPropertyDetailsController.scala

```diff
@@ -48,7 +48,7 @@ class CheckForeignPropertyDetailsController @Inject()(val authenticate: Authenti
   extends ClientConfirmedController with FeatureSwitching with I18nSupport {
 
   def show(origin: Option[String] = None): Action[AnyContent] = Action {
-    Ok("Check Foreign Property Details - WIP")
+    Redirect(controllers.incomeSources.add.routes.ForeignPropertyReportingMethodController.show("XAIS00000000004"))
```

Comment:
as discuss with QAs, we need to read incomeSourceId from the query string parameters , id key